### ISSUE TITLE
Login not possible

### ISSUE THREAD

**Issue by gerryvel** [tine20/tine20#6936]
After Update from 2018.08.2 to 2018.08.4 it's no poss. to login
I see:
"Anmeldung erfolgreich. Tine 2.0 ® wird geladen..."
no changes...
Ideas?

**Comment by msphn** [comment]
Check logfile and browser console.

**Comment by gerryvel** [comment]
Login is ok, but now is in different (10sec ... 5min) times an "auto-logout" activ.
"Ihre Sitzung ist abgelaufen. Bitte melden Sie sich erneut an." Browser problem?

**Comment by msphn** [comment]
Logfile please! And check error console in browser!

**Comment by tvannahl** [comment]
Got the same problem only for one user. No other user reported problems so far.

# Setting

 * Happend after update from 2018.08.2 to 2018.08.4
 * PHP 7.1
 * httpd 2.4
 * self build using `php composer.phar …` and `./vendor/bin/phing`.
 * custom patches not included

I was unable to find anything interesting in the logs, neither in the firefox console nor the server logs.

# Firefox Logs

![bildschirmfoto von 2018-10-17 16-34-26](https://user-images.githubusercontent.com/662132/47093967-8ebcd700-d22a-11e8-940b-8c4dccf31fd7.png)

```
RecordPickerManager::registerItem: TinebaseContainer Tinebase-c6de60230041ff0c7cf3-FAT.js:1:367786
RecordPickerManager::registerItem: TinebaseTag Tinebase-c6de60230041ff0c7cf3-FAT.js:1:367786 
```

# Server Logs

[tine20.log](https://github.com/tine20/tine20/files/2487891/tine20.log)


# Firefox Logs when launching application

The logs above are taken directly after login. These logs are taken on accessing `/tine20/index.php/Felamimail`.

![bildschirmfoto von 2018-10-17 16-37-28](https://user-images.githubusercontent.com/662132/47094185-00952080-d22b-11e8-9ca7-504a6b94c2b3.png)
```
SyntaxError: expected expression, got '<'[Weitere Informationen] ext-base.js:1
SyntaxError: expected expression, got '<'[Weitere Informationen] ext-all.js:1
[Mitteilungsdetails anzeigen/ausblenden] ReferenceError: Ext is not defined[Weitere Informationen] index.php:1:18286
Error: "Loading chunk 0 failed."
ahttps://www.example.de/tine20/Tinebase/js/fatClient.js-e678e4a87a75501f6f82-FAT.js:1:783 fatClient.js-e678e4a87a75501f6f82-FAT.js:1:3001
SyntaxError: expected expression, got '<'[Weitere Informationen] Tinebase-c6de60230041ff0c7cf3-FAT.js:1 
```

# Server logs when launching application

The logs above are taken directly after login. These logs are taken on accessing `/tine20/index.php/Felamimail`.

[tine20.log](https://github.com/tine20/tine20/files/2487915/tine20.log)

**Comment by tvannahl** [comment]
I got additional information regarding that problem:

It seems that everything is working for users that have an administrative role.

**Comment by msphn** [comment]
Probably you took away any mandatory role. Sadly there are no constraints for some circumstances.
Make sure your users are allowed to run Tinebase, Filemanager and so on.

**Comment by msphn** [comment]
The debug Log of Tine 2.0 should tell you more. You can enable it in setup.php!

**Comment by tvannahl** [comment]
> The debug Log of Tine 2.0 should tell you more. You can enable it in setup.php!

The logs you see are recorded while the development logging was enabled.

> Probably you took away any mandatory role. Sadly there are no constraints for some circumstances.
> Make sure your users are allowed to run Tinebase, Filemanager and so on.

Our user role has executable rights checked on every plugin without for **Admin**.

**Comment by tvannahl** [comment]
> I'm wondering about this line:
> 
> > e30e0 mmustermann - 2018-10-17T14:37:21+00:00 NOTICE (5): Tinebase_Frontend_Http::Tinebase_Frontend_Http::_deliverChangedFiles (404) File ./Expressomail/js/Expressomail-lang-de.js does not exist
> 
> do you have Expressomail installed? maybe you should remove that and try again.

Would happy to do that, I just don't know how. The administrators admin app gives me:
![bildschirmfoto von 2018-10-25 14-14-17](https://user-images.githubusercontent.com/662132/47499568-8daa2c00-d860-11e8-9a89-713e4d89dfe7.png)
So it's disabled but not uninstalled.

And the admin in *setup.php* doesn't see expressomail, hence I am unable to uninstall the application:
![bildschirmfoto von 2018-10-25 14-17-12](https://user-images.githubusercontent.com/662132/47499658-cb0eb980-d860-11e8-99e4-72cf90ee45f1.png)

**Comment by pschuele** [comment]
it may be possible to install the app via cli. please try this:

```
php setup.php --config=/path/to/config.inc.php --uninstall Expressomail
```

be careful: "--uninstall" without a param installs everything :smiling_imp:

**Comment by paulmhh** [comment]
@tvannahl on 2018.08.4 I can reproduce the issue regarding Admin/execute

it is fixed on 2018.08.5, I was not able to reproduce it there. So I close the issue, reopen or create a new issue if it is not resolved for you, thanks
